docs/docs/guides/integrate/login/saml.md

```diff
@@ -0,0 +1,284 @@
+
+```xml
+
+<?xml version="1.0" encoding="utf-8"?>
+<ns0:AuthnRequest
+	xmlns:ns0="urn:oasis:names:tc:SAML:2.0:protocol" ID="id-8LjuzBEUQFYFWjL55" Version="2.0" IssueInstant="2024-06-11T04:13:52Z" Destination="https://my-instance-xtzfbc.zitadel.cloud/saml/v2/SSO" AssertionConsumerServiceURL="http://127.0.0.1:5000/acs">
+	<ns1:Issuer
+		xmlns:ns1="urn:oasis:names:tc:SAML:2.0:assertion" Format="urn:oasis:names:tc:SAML:2.0:nameid-format:entity">https://zitadel-test.sp/metadata
+	</ns1:Issuer>
+	<ns2:Signature
+		xmlns:ns2="http://www.w3.org/2000/09/xmldsig#">
+		<ns2:SignedInfo>
+			<ns2:CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"></ns2:CanonicalizationMethod>
+			<ns2:SignatureMethod Algorithm="http://www.w3.org/2000/09/xmldsig#rsa-sha1"></ns2:SignatureMethod>
+			<ns2:Reference URI="#id-8LjuzBEUQFYFWjL55">
+				<ns2:Transforms>
+					<ns2:Transform Algorithm="http://www.w3.org/2000/09/xmldsig#enveloped-signature"></ns2:Transform>
+					<ns2:Transform Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"></ns2:Transform>
+				</ns2:Transforms>
+				<ns2:DigestMethod Algorithm="http://www.w3.org/2000/09/xmldsig#sha1"></ns2:DigestMethod>
+				<ns2:DigestValue>WSz/EQ72RZ0DTh3DBSRCElpITqM=</ns2:DigestValue>
+			</ns2:Reference>
+		</ns2:SignedInfo>
+		<ns2:SignatureValue>HHjsNh0OLj7...</ns2:SignatureValue>
+		<ns2:KeyInfo>
+			<ns2:X509Data>
+				<ns2:X509Certificate>MIIDtzCCAp+gAwIBAgIUfITRQGue...</ns2:X509Certificate>
+			</ns2:X509Data>
+		</ns2:KeyInfo>
+	</ns2:Signature>
+</ns0:AuthnRequest>
+```
+
+In this SAML request:
+
+- The **ID** (id-8LjuzBEUQFYFWjL55) is generated by the service provider, and the identity provider will respond to this ID in the SAML response.
+- **IssueInstant** (2024-05-11T04:13:52Z) is the timestamp for this request, used by the identity provider for verification to ensure the request is within an acceptable time frame.
+- **Destination** (https://my-instance-xtzfbc.zitadel.cloud/saml/v2/SSO) points to the identity provider's URL, ensuring the request is sent to the correct recipient.
+- **AssertionConsumerServiceURL** (http://127.0.0.1:5000/acs) indicates where the response should be sent after user authentication.
+- **Issuer** (https://zitadel-test.sp/metadata) is a predefined string formatted as a URL, matching what the identity provider expects.
+- **Signature** ensures the integrity and authenticity of the SAML request. It includes:
+- **SignedInfo** contains details about the canonicalization and signature methods.
+- **Reference** points to the signed data and includes a transform and digest method.
+- **SignatureValue** provides the actual digital signature.
+- **KeyInfo** containing the X.509 certificate, which holds the public key used to verify the signature.
+
+These XMLs are stringified, encoded, and passed in the query string to the identity provider, where the user is authenticated, and the SAML response is prepared, which will be received by the SP as an encoded string. 
```

Comment:
What do you mean as query string? If you mean as URL parameters, then maybe a comment to the different Bindings would be necessary as well, as the request is send as parameter or request body related to the bindings.

Reply:
It's also dependend on the binding, as to how the signature is sent.

Reply:
```suggestion
These XMLs are stringified, encoded, and transmitted according to the SAML binding used. For instance, in HTTP Redirect Binding, the request is sent as URL parameters, while in HTTP POST Binding, it is included in the request body. The signature's transmission method also depends on the binding used, ensuring the integrity and authenticity of the SAML message across different communication channels.
```